src/picologging/handlers.py

```diff
@@ -525,14 +525,61 @@ def __init__(self, queue):
         super().__init__()
         self.queue = queue
 
+    def enqueue(self, record):
+        """
+        Enqueue a record.
+
+        The base implementation uses put_nowait. You may want to override
+        this method if you want to use blocking, timeouts or custom queue
+        implementations.
+        """
+        self.queue.put_nowait(record)
+
+    def prepare(self, record):
```

Comment:
Please can you add a test that creates a bunch of `LogRecord` instances and uses this method, then compares the copied records to make sure all the attributes are the same value

Reply:
Added a hypothesis test, and in doing so, I discovered a slightly incompatibility with the case of passing in an empty string for stack_info. I don't know if that ever happens in the real world, but I had to make a small C change in order to bring the code in agreement with the standard library logic.